src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -947,23 +1214,34 @@ void doContainer(DocumentManagement documentManagement,
     responseWriter.finish();
   }
 
-  @VisibleForTesting
-  Node getNode(DocumentManagement documentManagement,
-      OpentextDocId opentextDocId) {
-    log.log(Level.FINER, "Looking up docId: " + opentextDocId);
-
+  private Node getNodeById(DocumentManagement documentManagement, long id) {
     // Use the object id to look up the node.
     Node node;
     try {
-      node = documentManagement.getNode(opentextDocId.getNodeId());
+      node = documentManagement.getNode(id);
       if (node == null) {
+        log.log(Level.FINER, "No item for id: " + id);
         return null;
       }
     } catch (SOAPFaultException soapFaultException) {
-      log.log(Level.WARNING, "Error retrieving item: " + opentextDocId,
+      log.log(Level.WARNING, "Error retrieving item: " + id,
           soapFaultException);
       return null;
     }
+    return node;
+  }
+
+  @VisibleForTesting
+  Node getNode(DocumentManagement documentManagement,
+      OpentextDocId opentextDocId) {
+    log.log(Level.FINER, "Looking up docId: " + opentextDocId);
+
+    // Use the object id to look up the node.
+    Node node = getNodeById(documentManagement, opentextDocId.getNodeId());
+    if (node == null) {
+      log.log(Level.FINER, "No item for id: " + opentextDocId);
```

Comment:
Logging it here lets the log message contain the full doc id (path) instead of just the numeric id in the cases where we're calling getNode with that value.